EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
+QUEST_LV_0300_20160310_001338	At first, I was thinking whether I should quit fishing, but... it soon turned out alright.{nl}I've earned a lot of money by picking up corals. {nl}
+QUEST_LV_0300_20160310_001339	The monsters were without a doubt a bit scary, but the lord was there to help us.{nl}The lord sent her army and lieutenant in exchange for a bit of offerings.{nl}
+QUEST_LV_0300_20160310_001340	But not too long ago, the monsters went crazy.{nl}We couldn't pick up the corals and even had to step out.{nl}
+QUEST_LV_0300_20160310_001341	Of course, we have told the Lord Lieutenant about this.{nl}However, he said protecting the town and defeating the monsters {nl}were two different things and the latter would cost additional offerings.{nl}
+QUEST_LV_0300_20160310_001342	We've ran out of the corals that we've collected. {nl}The merchants are waiting for us... What should we do...{nl} 
+QUEST_LV_0300_20160310_001343	I am just saying it is.{nl}Thank you for listening to the complaints of an old man.
+QUEST_LV_0300_20160310_001344	Hmm... Are you the rumored Revelator?{nl}I've heard you possess quite some talented skills.{nl}
+QUEST_LV_0300_20160310_001345	Have you heard of the coral named Lhu'uvik?{nl}It's a valuable coral used for creating magical tools or for finding magic principles.
+QUEST_LV_0300_20160310_001346	I have come all the way here to trade it...{nl}As you probably see, I am in trouble.{nl}
+QUEST_LV_0300_20160310_001347	There is nothing more important than the schedules and the promises of a merchant.{nl}If we go empty-handed like this, we will suffer from tremendous damage.{nl}
+QUEST_LV_0300_20160310_001348	It's not only us, but it also counts for those villagers.{nl}If it continues like this, they will lose their jobs and {nl}wander around looking for other places to live in.{nl}
+QUEST_LV_0300_20160310_001349	I want to borrow your power to help those poor villagers.{nl}Can you help me so that they can start picking up the corals again?
+QUEST_LV_0300_20160310_001350	Thank you for accepting.{nl}First, that lieutenant doesn't want to receive anything besides the corals for tributes.
+QUEST_LV_0300_20160310_001351	Eventually, we will have to get rid of the monsters so that the villagers can pick up the corals.{nl}You are the only one I can trust, so I ask you to get rid of it.{nl}
+QUEST_LV_0300_20160310_001352	I hope this clears up...{nl}In case something happens, while you are taking care of it, {nl}I will send a person to look at the status of the monsters.{nl}
+QUEST_LV_0300_20160310_001353	It will be great if we could just drive out the monsters, but...{nl}Something feels uncomfortable.
+QUEST_LV_0300_20160310_001354	Oh, you've come back already?{nl}You are as great as I've heard.
+QUEST_LV_0300_20160310_001355	I knew it. I knew that it wouldn't solve that easily.{nl}It's so strange that many monsters are rushing in within a short time.
```

Comment:
"I knew it wouldn't solve that easily." -> "I knew it wouldn't be that easy."

"rushing in within a short time." -> "rushing in so quickly."